### DISCUSSION TITLE
好后悔

### DISCUSSION THREAD

**Discussion by oh-holy-mom** [share121/inter-knot#2893] [其他]
md，大学前买电脑，在线下店买了一个六千多的华为触屏本，现在感觉亏死了

**Comment by CJHXAMSTLP** [comment]
二手卖了，亏不了太多的

**Comment by gold-shooter** [comment]
![Screenshot 2023-09-19 103427](https://github.com/user-attachments/assets/e0e44a17-1a94-4fa0-a2e1-41be13171ec2)
看你需求了，单纯用word工作软件的话买个轻薄本就行，要打游戏或者渲染软件这种高硬件要求的话买游戏本，当然，如果可以有条件弄台式机和用台式机的话就弄台式机，比笔记本钱少性能还好，就是要买的东西多
